### ISSUE TITLE
seems strange - shouldn't you return bool val in this case ?

### ISSUE THREAD

**Issue by melchior-by** [cyborg-shop/cucumber-training#6]
https://github.com/cyborg-shop/cucumber-training/blob/390187c19f0592138f232b2b7ebdd9e11616c1a6/src/main/java/pages/wikipedia/ResultPage.java#L13

**Comment by cyborg-shop** [comment]
> https://github.com/cyborg-shop/cucumber-training/blob/390187c19f0592138f232b2b7ebdd9e11616c1a6/src/main/java/pages/wikipedia/ResultPage.java#L13

Yeah.. replaced with bool. Also I added assert in test area for this text searching
